src/create-config.ts

```diff
@@ -0,0 +1,322 @@
+import { deriveEndpointId } from '@api3/airnode-admin';
+import { ChainConfig, NodeSettings, Triggers, Trigger, ApiCredentials, Config } from '@api3/airnode-node';
+import {
+  AirkeeperConfig,
+  AirkeeperChainConfig,
+  Triggers as AirkeeperTriggers,
+  Subscriptions,
+  Templates as AirkeeperTemplates,
+  Endpoints as AirkeeperEndpoints,
+} from '@api3/airkeeper/dist/src/validator';
+import { OperationsRepository } from './types';
+import { promptQuestions, readOperationsRepository, writeOperationsRepository } from './utils/filesystem';
+import { runAndHandleErrors } from './utils/cli';
+import { chainNameToChainId, DapiServerInterface } from './utils/evm';
+
+const questions = (choices: Choice[]): PromptObject[] => {
+  return [
+    {
+      type: 'autocomplete',
+      name: 'apiName',
+      message: 'What is the name of the API Integration?',
+      choices: choices,
+    },
+    {
+      type: 'confirm',
+      name: 'airnodeAuthorizer',
+      message: 'Do you want to include the airnode authorizer?',
+      initial: false,
+    },
+    {
+      type: 'confirm',
+      name: 'airnodeHeartbeat',
+      message: 'Do you want to enable the airnode heartbeat?',
+      initial: false,
+    },
+  ];
+};
+
+const main = async () => {
+  const operationsRepository = readOperationsRepository();
+  const apiChoices = Object.keys(operationsRepository.apis).map((api) => ({ title: api, value: api })) as Choice[];
+  const response = await promptQuestions(questions(apiChoices));
+  const apiData = operationsRepository.apis[response.apiName];
+  const secrets = [];
+
+  //// Build Config.json  ////
+
+  // Get all the chains the API will be deployed on
+  const apiChains = [
+    ...new Set(
+      Object.keys(apiData.beacons).flatMap((beacon) => apiData.beacons[beacon].chains.map((chain) => chain.name))
+    ),
+  ];
+
+  const chains: ChainConfig[] = apiChains.map((chainName) => {
+    const chainId = chainNameToChainId[chainName];
+    //TODO: Add RequesterAuthorizerWithManager
+    const authorizers =
+      response.airnodeAuthorizer && RequesterAuthorizerWithAirnodeAddresses[chainId]
+        ? [RequesterAuthorizerWithAirnodeAddresses[chainId]]
+        : [];
+    const contracts = { AirnodeRrp: AirnodeRrpAddresses[chainId] || '' };
+    const options = {
+      txType: 'eip1559' as const,
+      priorityFee: {
+        value: 3.12,
+        unit: 'gwei' as const,
+      },
+      baseFeeMultiplier: 2,
+    };
+    const providers = {
+      provider1: {
+        url: `\${${chainName}_PROVIDER_URL}`.toUpperCase(),
+      },
+    };
+
+    // push the provider url to the secrets
+    secrets.push(`${chainName}_PROVIDER_URL=`.toUpperCase());
+
+    return {
+      authorizers,
+      contracts,
+      id: `${chainId}`,
+      providers,
+      type: 'evm',
+      options,
+      maxConcurrency: 100,
+      blockHistoryLimit: 300,
+      minConfirmations: 0,
+    };
+  });
+
+  const nodeSettings: NodeSettings = {
+    nodeVersion: '0.5.0',
```

Comment:
You can do this:
Maybe something like this:
```suggestion
  const nodeVersion = require('@api3/airnode-node/package.json').version;

  const nodeSettings = {
    nodeVersion,
```